lib/rules/key-spacing.js

```diff
@@ -333,11 +336,18 @@ module.exports.schema = [
             "align": {
                 "enum": ["colon", "value"]
             },
+            "mode": {
+                "enum": ["strict", "minimum"]
+            },
             "beforeColon": {
-                "type": "boolean"
+                "type": ["boolean", "number"],
```

Comment:
Okay, reverted. I guess we need a new option to pass the indent size for aligning to that. `snapTo`?